server/models/Collective.js

```diff
@@ -1800,16 +1800,14 @@ export default function (Sequelize, DataTypes) {
       throw new Error('This host is not open to applications');
     }
 
+    const platformFeePercent = this.tags?.includes('covid-19') ? 0 : hostCollective.platformFeePercent;
```

Comment:
What about `EVENT` and `PROJECT`? How do we make sure they inherit their parent?